src/Sulu/Bundle/ContentBundle/Preview/ContextParametersNotFoundException.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/*
+ * This file is part of the Sulu CMF.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+namespace Sulu\Bundle\ContentBundle\Preview;
+
+/**
+ * Represents a not started preview
+ */
+class ContextParametersNotFoundException extends \Exception
```

Comment:
What is a context parameter? According to the doc block this should probably be called `PreviewNotStartedException`